testing/constraints-3.9.txt

```diff
@@ -12,6 +12,7 @@ google-cloud-resource-manager==1.10.3
 google-cloud-storage==2.0.0
 ibis-framework==8.0.0
 pandas==1.5.0
+pyarrow==3.0.0
```

Comment:
can this be higher? looks like 
E   AttributeError: type object 'pyarrow.lib.RecordBatch' has no attribute 'from_pydict'
is due to 3.0.0 being too old